### PR TITLE
nixos/doc: convert Chapter 7, 9, 10, 11, 12, 14, 15, 48 to CommonMark

### PR DESCRIPTION
**Pull request by bobby285271**
###### Motivation for this change

- [Related GitHub project](https://github.com/NixOS/nixpkgs/projects/37)
- [Related Discourse post](https://discourse.nixos.org/t/happy-docsgiving/10214/28)

###### Things done

Do conversion of eight files from DocBook to CommonMark per [RFC 72](https://github.com/NixOS/rfcs/blob/master/rfcs/0072-commonmark-docs.md).

- `nixos/doc/manual/configuration/user-mgmt.xml`
- `nixos/doc/manual/configuration/x-windows.xml`
- `nixos/doc/manual/configuration/wayland.xml`
- `nixos/doc/manual/configuration/gpu-accel.xml`
- `nixos/doc/manual/configuration/xfce.xml`
- `nixos/doc/manual/configuration/linux-kernel.xml`
- `nixos/doc/manual/configuration/subversion.xml`
- `nixos/doc/manual/configuration/kubernetes.xml`

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/doc/manual/configuration/gpu-accel.chapter.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+the `OCL_ICD_VENDORS` environment variable. This variable can contain a
+directory which is scanned by the ICL loader for ICD files. For example:
+
+```ShellSession
+$ export \
+  OCL_ICD_VENDORS=`nix-build '<nixpkgs>' --no-out-link -A rocm-opencl-icd`/etc/OpenCL/vendors/
+```
+
+The second mechanism is to add the OpenCL driver package to
+[`hardware.opengl.extraPackages`](options.html#opt-hardware.opengl.extraPackages).
```
Comment (jtojnar) [inline nixos/doc/manual/configuration/gpu-accel.chapter.md:33]: Options need to be linked using just the URI fragment or they will not be checked.

Comment (jtojnar) [reply nixos/doc/manual/configuration/gpu-accel.chapter.md:33]: Also, you can use automatic links when you omit the link text.

File: nixos/doc/manual/configuration/gpu-accel.chapter.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+Next](https://en.wikipedia.org/wiki/Graphics_Core_Next) (GCN) GPUs are
+supported through either radv, which is part of mesa, or the amdvlk
+package. Adding the amdvlk package to
+[`hardware.opengl.extraPackages`](options.html#opt-hardware.opengl.extraPackages)
+makes amdvlk the default driver and hides radv and lavapipe from the device list.
+A specific driver can be forced as follows:
+
+```nix
+hardware.opengl.extraPackages = [
+         pkgs.amdvlk
```
Comment (jtojnar) [inline nixos/doc/manual/configuration/gpu-accel.chapter.md:147]: Indentation is off here.

**Comment by vcunat** [comment]
Do you have a nice way of dealing with conflicts?  On current `staging-next` we have changes in `nixos/doc/manual/configuration/linux-kernel.xml`

**Comment by vcunat** [comment]
If it's (almost) automatic, perhaps redoing the same on the `staging-next` version of the XML?

**Comment by bobby285271** [comment]
@vcunat I think we can:

- On `master` branch, do this: https://github.com/bobby285271/nixpkgs/commit/45b2c68cc38009b77af3442cc968f591349de27c (probably needs someone other than me to check building the NixOS manual)
- Attempt to merge `master` into `staging-next`
- Delete `nixos/doc/manual/configuration/linux-kernel.xml`

**Pull request merged**